rfd/0109-opsgenie-plugin.md

```diff
@@ -0,0 +1,176 @@
+# Opsgenie Integration RFD
```

Comment:
Nit: Use the same title format as all other RFDs.
```suggestion
# RFD 109: Opsgenie Integration
```

Btw, is it Opsgenie or OpsGenie? Very minor and purely cosmetic, but can we please make sure we're using one or the other consistently throughout the document?